core/templates/dev/head/components/CkEditorRteDirective.js

```diff
@@ -96,19 +96,17 @@ oppia.directive('ckEditorRte', [
         // a self created plugin (other plugins are provided by CKEditor).
         CKEDITOR.plugins.addExternal(
           'pre', '/extensions/ckeditor_plugins/pre/', 'plugin.js');
-        CKEDITOR.plugins.addExternal(
-          'indentblock',
-          '/third_party/static/ckeditor-indentblock-4.9.2/', 'plugin.js');
         CKEDITOR.plugins.addExternal(
           'blockquote',
           '/third_party/static/ckeditor-blockquote-4.9.2/', 'plugin.js');
 
         // Initialize CKEditor.
         var ck = CKEDITOR.inline(el[0].children[0].children[1], {
           extraPlugins:
-          'indentblock,pre,blockquote,widget,lineutils,sharedspace,' +
+          'pre,blockquote,widget,lineutils,sharedspace,' +
           pluginNames,
           startupFocus: true,
+          removePlugins: 'indentblock',
```

Comment:
Why do we need to have this here if we've already removed it from extraPlugins?

Reply:
We need this because CKEditor full automatically adds both indentblock and indentlist plugins. I got a list of all plugins which are added in CKEditor config by doing a `console.log(CKEDITOR.config.plugins)`. It includes `indent, indentblock, indentlist`. So, without removing the indentblock plugin we cannot disable indentation for lists.

Also even if I had not added indentblock plugin as extra plugin, indentation will work for both blocks and lists.

Reply:
Ah ok. But there's one thing I don't understand, then -- why did we need to add the indentblock plugin manually before this?

Reply:
I did not know this before, I tried CKEditor demo online and saw that indent was disabled for blocks, so I added this plugin but after looking into this now, I found out that the latest version of CKEditor does not require this since it is already added to build-config.

Reply:
OK, thanks -- that's a good find. Did you check that all other extraPlugins are still required?

Reply:
I just checked this - This is the complete list
```
dialogui,dialog,about,a11yhelp,dialogadvtab,basicstyles,bidi,blockquote,notification,
button,toolbar,clipboard,panelbutton,panel,floatpanel,colorbutton,colordialog,
templates,menu,contextmenu,copyformatting,div,resize,elementspath,
enterkey,entities,popup,filetools,filebrowser,find,fakeobjects,flash,
floatingspace,listblock,richcombo,font,forms,format,horizontalrule,
htmlwriter,iframe,wysiwygarea,image,indent,indentblock,indentlist,smiley,
justify,menubutton,language,link,list,liststyle,magicline,maximize,newpage,
pagebreak,pastetext,pastefromword,preview,print,removeformat,
save,selectall,showblocks,showborders,sourcearea,specialchar,scayt,
stylescombo,tab,table,tabletools,tableselection,undo,lineutils,
widgetselection,widget,notificationaggregator,uploadwidget,uploadimage,wsc
```
I have removed widget, lineutils and blockquote as well